test/testcondition.cpp

```diff
@@ -3336,7 +3336,7 @@ class TestCondition : public TestFixture {
               "    const int b = 52;\n"
               "    return a+b;\n"
               "}");
-        ASSERT_EQUALS("", errout.str());
+        ASSERT_EQUALS("[test.cpp:4]: (style) Condition 'a+b' is always true\n", errout.str());
```

Comment:
Not directly releated to this PR, but shouldnt we better state: return value is always true?

Reply:
Yea that needs to be updated too, I plan on a later PR to update that.